components/EmbeddedDashboard.tsx

```diff
@@ -83,11 +86,15 @@ const componentPageDisplayName: Partial<Record<ComponentPage, string>> = {
 };
 
 export const EmbeddedDashboardInternal: React.FC<Props> = (props) => {
+  const [featuresConfig, setFeaturesConfig] = React.useState<
```

Comment:
The UI looks great!

Instead of this being a feature that changes "on the fly", my recommendation is to make this a feature that is stored in local storage, similar to locale: https://github.com/jogonzal/jorgeconnectplatform/blob/main/clientsStorage/LocalStorageEntry.ts#L62. I think that can make the implementation simpler as well and actually very similar to what you have:

1.  We have a dialog very similar to the one you have that is invoked from a "settings" button, maybe at the bottom right of the page, similar to ConnectJSSource
![image](https://github.com/jogonzal/jorgeconnectplatform/assets/95381655/0b96bbd2-9ec8-42c0-9396-fbb1b608c153)
2. This dialog has a "save" and a "reset" button
3. When you save, it stores to local storage
4. When running this `useConnectJSInit` code, you check local storage before making the API call to create account session

Generally that is how other settings work in connect-test, I'd like to keep it that way (that way you get these persisting even after refresh)

Reply:
Agreed, this approach is much more straightforward!